src/vcpkg/base/downloads.cpp

```diff
@@ -341,7 +341,7 @@ namespace vcpkg
             if (!m_hSession.Open(context,
                                  sanitized_url,
                                  L"vcpkg/1.0",
-                                 WINHTTP_ACCESS_TYPE_NO_PROXY,
+                                 WINHTTP_ACCESS_TYPE_AUTOMATIC_PROXY,
```

Comment:
As I described in the issue submission, I believe this introduces unacceptable divergence between the curl backend and the WinHTTP backend.